pkg/clients/ocm/client.go

```diff
@@ -10,20 +10,35 @@ import (
 type Environment string
 
 const (
-	Production  Environment = "https://api.openshift.com"
-	Stage       Environment = "https://api.stage.openshift.com"
-	Integration Environment = "https://api.integration.openshift.com"
+	Production         Environment = "https://api.openshift.com"
+	Stage              Environment = "https://api.stage.openshift.com"
+	Integration        Environment = "https://api.integration.openshift.com"
+	FedRampProduction  Environment = "https://api.openshiftusgov.com"
+	FedRampStage       Environment = "https://api.stage.openshiftusgov.com"
+	FedRampIntegration Environment = "https://api.int.openshiftusgov.com"
+	FRTokenURL         string      = "https://sso.int.openshiftusgov.com/realms/redhat-external/protocol/openid-connect/token"
```

Comment:
```suggestion
	fedrampTokenURL         string      = "https://sso.int.openshiftusgov.com/realms/redhat-external/protocol/openid-connect/token"
```
this can be made private